src/test/util.ts

```diff
@@ -247,6 +247,17 @@ export class TestClient {
         CompletionRequest.type, params) as Promise<CompletionList>;
   }
 
+  async getReferences(
+      path: string, position: SourcePosition,
+      includeDefinition = false): Promise<Location[]> {
```

Comment:
could you add a test that uses `includeDefinition = true`?

Reply:
+1, done